lib/fibered_mysql2.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+require 'async'
 require 'fibered_mysql2/version'
```

Comment:
(nit) Its more cleaner to have it as follows:

```
require_relative 'active_record/connection_adapters/fibered_mysql2_adapter'
require_relative 'fibered_mysql2/fibered_database_connection_pool'
require_relative 'fibered_mysql2/fibered_mutex_with_waiter_priority'
require_relative 'fibered_mysql2/fibered_mysql2_connection_factory'
require_relative 'fibered_mysql2/version'
```

Reply:
Agreed! Fixed: https://github.com/Invoca/fibered_mysql2/pull/9/commits/910b8bc095daf3136e7a2c167035d72f2ea9c663